pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -169,10 +170,10 @@ func TestPriorityQueue_Add(t *testing.T) {
 type fakeFramework struct{}
 
 func (*fakeFramework) QueueSortFunc() framework.LessFunc {
-	return func(podInfo1, podInfo2 *framework.PodInfo) bool {
-		prio1 := podutil.GetPodPriority(podInfo1.Pod)
-		prio2 := podutil.GetPodPriority(podInfo2.Pod)
-		return prio1 < prio2
+	return func(pInfo1, pInfo2 *framework.PodInfo) bool {
+		p1 := pod.GetPodPriority(pInfo1.Pod)
+		p2 := pod.GetPodPriority(pInfo2.Pod)
+		return (p1 > p2) || (p1 == p2 && pInfo1.Timestamp.Before(pInfo2.Timestamp))
```

Comment:
The name "LessFunc" denotes the default heap behavior - minHeap, the object which is "less" than the other is put ahead of the heap. However, in our PriorityQueue, it's maxHeap, so the "LessFunc" should return true if p1 > p2. The old test logic isn't correct.

I've also corrected the testcase at `TestPriorityQueue_AddWithReversePriorityLessFunc `.